pkg/k8s/client_test.go

```diff
@@ -535,3 +538,99 @@ func Test_GetValidatingWebHookConfiguration(t *testing.T) {
 		})
 	}
 }
+
+func Test_GetCRD(t *testing.T) {
+	t.Parallel()
+
+	// define test cases
+	testCases := []struct {
+		name              string
+		givenCRDName      string
+		wantNotFoundError bool
+	}{
+		{
+			name:              "should return not found error when CRD is missing in k8s",
+			givenCRDName:      ApplicationCrdName,
+			wantNotFoundError: false,
+		},
+		{
+			name:              "should return correct CRD from k8s",
+			givenCRDName:      "non-existing",
+			wantNotFoundError: true,
```

Comment:
I think the names of the test cases are switched.

Reply:
updated